src/main/java/mate/academy/exception/AuthenticationException.java

```diff
@@ -1,8 +1,13 @@
 package mate.academy.exception;
 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
 public class AuthenticationException extends Exception {
+    private static final Logger logger = LogManager.getLogger(AuthenticationException.class);
```

Comment:
Please remove logger from Exception class. We log error during handling of exceptions, not during creation